### PR TITLE
Psd 262 lend filter frame setup

### PR DESCRIPTION
**Pull request by dymonda21**
Created a page titled Lend Filter Alpha and imported the Wwwpage component.
Not testable yet. 

https://kiva.atlassian.net/browse/PSD-262

### CODE REVIEW

**Comment by go-bears** [comment]
first PR! 🎉  congrats @dymonda21 !

**Review by meganmcmillan — APPROVED** [review]
Congratulations! 🎉 🥇

File: src/pages/Lend/FilterAlpha/LendFilterAlpha.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	>
+		<article class="tw-relative">
+			<div class="tw-relative" style="max-height: 460px;">
+				<div class="tw-absolute tw-top-0 tw-h-full tw-w-full tw-overflow-hidden">
+				</div>
+			</div>
+			<div class="lg:tw-absolute lg:tw-w-full lg:tw-h-full lg:tw-top-0 lg:tw-pt-8">
+			</div>
+		</article>
+		<!-- <aside>Similar loans</aside> -->
```
Comment (mcstover) [inline src/pages/Lend/FilterAlpha/LendFilterAlpha.vue:15]: can be removed in future work

File: src/pages/Lend/FilterAlpha/LendFilterAlpha.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+export default {
+	components: {
+		WwwPage,
+	},
+	data() {
+		return {
+			headerTheme: lightHeader,
+			loanId: Number(this.$route.params.id || 0),
```
Comment (mcstover) [inline src/pages/Lend/FilterAlpha/LendFilterAlpha.vue:31]: `loanId` is also unnecessary in the context and can be removed in a future pr.

**Review by mcstover — APPROVED** [review]

**Comment by go-bears** [comment]
🎉 !

**Pull request merged**